### ISSUE TITLE
LocalizationValidationScripts LocalizationValidiationScriptsTagHelperComponent wrong culture

### ISSUE THREAD

**Issue by FJGR65** [LazZiya/TagHelpers#16]
When "culture" is replaced in the following scripts, they cannot be downloaded because they do not exist.
This happens when using culture names such as "en-US" or "es-ES".
Only the first two characters should be selected: "en" or "es". For these cultures, if there are downloads.

````cshtml
<script type="text/javascript">
    $.when(
        $.get("https://cdn.jsdelivr.net/gh/unicode-cldr/cldr-core@{cldr-core-version}/supplemental/likelySubtags.json"),
        $.get("https://cdn.jsdelivr.net/gh/unicode-cldr/cldr-numbers-full@{cldr-core-version}/main/{culture}/numbers.json"),
        $.get("https://cdn.jsdelivr.net/gh/unicode-cldr/cldr-numbers-full@{cldr-core-version}/main/{culture}/currencies.json"),
        $.get("https://cdn.jsdelivr.net/gh/unicode-cldr/cldr-core@{cldr-core-version}/supplemental/numberingSystems.json"),
        $.get("https://cdn.jsdelivr.net/gh/unicode-cldr/cldr-dates-full@{cldr-core-version}/main/{culture}/ca-gregorian.json"),
        $.get("https://cdn.jsdelivr.net/gh/unicode-cldr/cldr-dates-full@{cldr-core-version}/main/{culture}/timeZoneNames.json"),
        $.get("https://cdn.jsdelivr.net/gh/unicode-cldr/cldr-core@{cldr-core-version}/supplemental/timeData.json"),
        $.get("https://cdn.jsdelivr.net/gh/unicode-cldr/cldr-core@{cldr-core-version}/supplemental/weekData.json")
    ).then(function () {
        // Normalize $.get results, we only need the JSON, not the request statuses.
        return [].slice.apply(arguments, [0]).map(function (result) {
            return result[0];
        });
    }).then(Globalize.load).then(function () {
        Globalize.locale("{culture}");
    });
</script>
`````

**Comment by LazZiya** [comment]
It seems that these two cultures (en-US and es-ES) are not available in [cldr-numbers](https://www.jsdelivr.com/package/npm/cldr-numbers-full?path=main) in jsdelivr network. This is not the case for all cultures, the majority of the cultures are available as two and four letter coded names.

Thanks @FJGR65 for submitting this issue, I will provide a hotfix soon.

**Comment by LazZiya** [comment]
The issue fixed in dd0c66ea but somehow I missed to close it in time.

Now the cultures that has no relevant script exists in the [`cldr-numbers`](https://www.jsdelivr.com/package/npm/cldr-numbers-full?path=main) will be trimmed to the parent culture. Currently two cultures are in the list `en-us` and `es-es`, more cultures to be added if necessary.

````cs
string[] nonSupportedCultres = { "en-us", "es-es" };
culture = nonSupportedCultres.Any(x => x.Equals(culture, StringComparison.OrdinalIgnoreCase))
        ? culture.Split('-')[0]
        : culture;
````
